<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="5" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-20T21:10:29 IST" name="Default suite" finished-at="2021-09-20T21:12:06 IST" duration-ms="96548">
    <groups>
    </groups>
    <test started-at="2021-09-20T21:10:29 IST" name="Default test" finished-at="2021-09-20T21:12:06 IST" duration-ms="96548">
      <class name="Trello.PostAssessment.TrelloBoard">
        <test-method is-config="true" signature="setup()[pri:0, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:10:29 IST" name="setup" finished-at="2021-09-20T21:10:40 IST" duration-ms="11070" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="login()[pri:1, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:10:40 IST" name="login" finished-at="2021-09-20T21:11:04 IST" duration-ms="23589" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Log in to continue - Log in with Atlassian account] but found [Boards | Trello]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Log in to continue - Log in with Atlassian account] but found [Boards | Trello]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Trello.PostAssessment.TrelloBoard.login(TrelloBoard.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="board()[pri:2, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:11:04 IST" name="board" finished-at="2021-09-20T21:11:50 IST" duration-ms="45767" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Boards | Trello] but found [Goa Trip Preparation | Trello]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Boards | Trello] but found [Goa Trip Preparation | Trello]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at Trello.PostAssessment.TrelloBoard.board(TrelloBoard.java:62)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- board -->
        <test-method signature="createDo()[pri:3, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:11:50 IST" name="createDo" finished-at="2021-09-20T21:11:52 IST" duration-ms="2780" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createDo -->
        <test-method signature="doing()[pri:4, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:11:52 IST" name="doing" finished-at="2021-09-20T21:11:57 IST" duration-ms="4630" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doing -->
        <test-method signature="done()[pri:5, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:11:57 IST" name="done" finished-at="2021-09-20T21:12:02 IST" duration-ms="4606" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- done -->
        <test-method signature="deleteBoard()[pri:6, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:12:02 IST" name="deleteBoard" finished-at="2021-09-20T21:12:03 IST" duration-ms="1011" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteBoard -->
        <test-method signature="logout()[pri:7, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:12:03 IST" name="logout" finished-at="2021-09-20T21:12:06 IST" duration-ms="2929" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="close()[pri:0, instance:Trello.PostAssessment.TrelloBoard@4b5a5ed1]" started-at="2021-09-20T21:12:06 IST" name="close" finished-at="2021-09-20T21:12:06 IST" duration-ms="74" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- Trello.PostAssessment.TrelloBoard -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
